tests/Command/RunCommandTest.php

```diff
@@ -1090,17 +1067,17 @@ public function testExecuteStateTransitionError(): void
             'command' => $command->getName(),
         ]);
 
+        self::assertFalse($testHandler->hasErrorRecords());
+        self::assertFalse($testHandler->hasCriticalRecords());
+        self::assertFalse($testHandler->hasWarningRecords());
+
+        self::assertTrue($testHandler->hasInfoThatContains(
+            'Running job "123".'
+        ));
         self::assertTrue($testHandler->hasNoticeThatContains(
             'Failed to save result for job "123". State transition forbidden:'
         ));
         self::assertEquals(0, $ret);
-
-        $events = $this->storageClient->listEvents(['runId' => '124']);
-        $messages = array_column($events, 'message');
-
-        self::assertNotEmpty($messages);
-        self::assertContains('Running job "123".', $messages);
-        self::assertNotContains('Failed to save result for job "123". State transition forbidden:', $messages);
```

Comment:
Tohle je imho zbytečný,když o kus výš validujeme že se to posílá v notice kanálu

Reply:
asi jo